bin/iotaJsonTester.js

```diff
@@ -135,33 +136,34 @@ function exitClient() {
 }
 
 var commands = {
-    'config': {
+    config: {
         parameters: ['host', 'port', 'apiKey', 'deviceId'],
         description: '\tConfigure the client to emulate the selected device, connecting to the given host.',
         handler: setConfig
     },
-    'showConfig': {
+    showConfig: {
         parameters: [],
         description: '\tConfigure the client to emulate the selected device, connecting to the given host.',
         handler: getConfig
     },
-    'connect': {
+    connect: {
         parameters: [],
         description: '\tConnect to the MQTT broker.',
         handler: connect
     },
-    'singleMeasure': {
+    singleMeasure: {
         parameters: ['attribute', 'value'],
         description: '\tSend the given value for the selected attribute to the MQTT broker.',
         handler: checkConnection(singleMeasure)
     },
-    'multipleMeasure': {
+    multipleMeasure: {
         parameters: ['attributes'],
-        description: '\tSend a collection of attributes to the MQTT broker, using JSON format. The "attributes"\n' +
```

Comment:
This line is 117 chars long. I wonder why prettier has changed it if printWidth is 120...

Reply:
Presumably prettier picks up on the `\n` and breaks at that point

Reply:
A bit weird... but ok.

NTC